CITATION.cff

```diff
@@ -0,0 +1,35 @@
+cff-version: 1.2.0
+message: 'If you use this software, please cite it as below.'
+title: Autometa
```

Comment:
```suggestion
```
Title is defined towards the bottom with actual paper's title so remove this one